src/go/k8s/pkg/console/console.go

```diff
@@ -36,11 +37,13 @@ type ConsoleConfig struct {
 	License    string          `json:"license,omitempty" yaml:"license,omitempty"`
 	Enterprise Enterprise      `json:"enterprise,omitempty" yaml:"enterprise,omitempty"`
 	Login      EnterpriseLogin `json:"login,omitempty" yaml:"login,omitempty"`
+	Cloud      CloudConfig     `json:"cloud,omitempty" yaml:"cloud,omitempty"`
 }
 
 // SetDefaults sets sane defaults
 func (cc *ConsoleConfig) SetDefaults() {
 	cc.Kafka.SetDefaults()
+	cc.Cloud.PrometheusEndpoint.SetDefaults()
```

Comment:
I think we should set these in the CRD instead

Reply:
I was just consistent with what you have but I can add it to defaulting webhook

Reply:
We don't have Kafka in the CRD